var/spack/repos/builtin/packages/yaml-cpp/package.py

```diff
@@ -16,10 +16,18 @@ class YamlCpp(CMakePackage):
     url      = "https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.5.3.tar.gz"
     git      = "https://github.com/jbeder/yaml-cpp.git"
 
+    def url_for_version(self, version):
```

Comment:
Can you move this down below with the other function definitions?

Reply:
Done.